bgpd/bgp_attr.c

```diff
@@ -4117,9 +4117,12 @@ bgp_size_t bgp_packet_attribute(struct bgp *bgp, struct peer *peer,
 			stream_putc(s, BGP_ATTR_FLAG_OPTIONAL
 					       | BGP_ATTR_FLAG_TRANS);
 			stream_putc(s, BGP_ATTR_PREFIX_SID);
-			stream_putc(s, 24);     /* tlv len */
+			stream_putc(s, 28); /* tlv len */
```

Comment:
Can we get defines for the 28 here and the 25( on line 4122) so that things are somewhat human readable?

Reply:
I updated the code to calculate 28 and 25 above this code.